feat(sentry): adding a check if sentry is available (#11596)
backstage/backstage: .changeset/fifty-dogs-collect.md

---
'@backstage/plugin-sentry': patch
---

> Providing a way for users to determine if sentry is available.

Reviewer: ```suggestion
Exported `isSentryAvailable` which can be used to determine if sentry is available.
```
